src/components/sets/CellSetsManagerSubscriber.js

```diff
@@ -145,7 +150,8 @@ export default function CellSetsManagerSubscriber(props) {
     const cellSetsAddToken = PubSub.subscribe(CELL_SETS_ADD,
       (msg, treeToImport) => {
         const actionType = (initEmit ? ACTION.IMPORT_AND_VIEW : ACTION.IMPORT);
-        dispatch({ type: actionType, levelZeroNodes: treeToImport.tree });
+        const newTreeToImport = tryUpgradeToLatestSchema(treeToImport, SETS_DATATYPE_CELL);
```

Comment:
Can you add a direct object, like `tryUpgradeTreeToLatestSchema`?  I had to look at the code to understand what it does.

Reply:
Yes